src/syngen/ml/data_loaders/data_loaders.py

```diff
@@ -86,7 +86,7 @@ def _load_data(path, **kwargs) -> Tuple[pd.DataFrame, Dict]:
         df = pd.DataFrame()
         try:
             df = pd.read_csv(path, engine="c", **kwargs).apply(trim_string, axis=0)
-            return df, CSVConvertor(df).schema
+            return df, CSVConvertor({"fields": {}, "format": "CSV"}, df).schema
```

Comment:
From my point of view, this string should be replaced with the previous one - return df, CSVConvertor(df).schema